### PR TITLE
Add server bind option

### PR DESCRIPTION
**Pull request by op69qs**
Add Server.path and Server.bindAddress two options to special http server handle path and Bind Address.

### CODE REVIEW

File: cmd/podsync/server.go [hunk 1]
```diff
@@ -18,14 +18,20 @@ func NewServer(cfg *config.Config) *Server {
...
+	bindAddress := cfg.Server.BindAddress
+	if bindAddress == "*" {
+		bindAddress = ""
+	}
 	srv := Server{}
 
-	srv.Addr = fmt.Sprintf(":%d", port)
-	log.Debugf("using address: %s", srv.Addr)
+	srv.Addr = fmt.Sprintf("%s:%d", bindAddress, port)
+	//srv.Addr = fmt.Sprintf("{{$bindAddress:$port}}")
```
Comment (mxpv) [inline cmd/podsync/server.go:28]: nit: remove this comment?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,10 @@ Here is an example how configuration might look like:
 ```toml
 [server]
 port = 8080
+# Bind a specific IP addresses for server ,"*": bind all IP addresses which is default option, localhost or 127.0.0.1  bind a single IPv4 address
+bind_address = "172.20.10.2" 
+# Specify path for reverse proxy and only [A-Za-z0-9]
+path = "test"
```
Comment (mxpv) [inline README.md:63]: Could you please update the comment that this is related to URL path specifically? Possibly with en example.
(I was confused for a moment why we need both `path` and `data_dir` until I referred to Go's documentation).

**Review by mxpv — COMMENTED** [review]
Thanks for the PR! A few suggestions below.

File: cmd/podsync/server.go (hunk 1 above)
Comment (op69qs) [reply cmd/podsync/server.go:28]: OK ,my bad miss that

File: README.md (hunk 2 above)
Comment (op69qs) [reply README.md:63]: got it

**Comment by Gyurmi80** [comment]
I WILL KILL YOU!

Idézet (Maksym Pavlenko <notifications@github.com>):

>  
>
>   Merged #190[1] into master.
>
>   —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub[2], or unsubscribe[3].



Hivatkozások:
-------------
[1] https://github.com/mxpv/podsync/pull/190
[2] https://github.com/mxpv/podsync/pull/190#event-3924008672
[3]

**Pull request merged**